### DISCUSSION TITLE
What does hiding Proposed objects do?

### DISCUSSION THREAD

**Discussion by ERYpTION** [osmandapp/OsmAnd#15975] [Q&A]
It is possible to hide 'Proposed objects' on the map. However, even when I select to do this in the map rendering settings I still see objects that I know are using the proposed key (https://wiki.openstreetmap.org/wiki/Key:proposed). What is this option hiding?

**Comment by vshcherb** [comment]
https://github.com/osmandapp/OsmAnd-resources/blob/master/rendering_styles/default.render.xml all implementation is here

**Comment by ERYpTION** [comment]
I would like to suggest adding landuse=greenfield as one of the 'Proposed objects' that can be hidden.
